src/azure-cli-core/HISTORY.rst

```diff
@@ -2,12 +2,12 @@
 
 Release History
 ===============
+cloud set: fix a bogus error about subscription not found 
```

Comment:
Nit: 2.0.60 is the next version to be released, so I'd replace "* Minor fixes." with the text you provided.